src/renderer/components/ReactFlowBox.tsx

```diff
@@ -343,6 +466,8 @@ const ReactFlowBox = memo(({ wrapperRef, nodeTypes, edgeTypes }: ReactFlowBoxPro
                     borderRadius: '0.5rem',
                 }}
                 onConnect={createConnection}
+                onConnectStart={onConnectStart}
+                onConnectStop={onConnectStop}
```

Comment:
Sorry for not being clear enough. I meant that the whole code you added to this component should be its own hook. From what I see, the new pane is separate from the react flow box. The RFB only really needs the callbacks used here and another used below. All other code could be moved into the new hook.

Reply:
Oooh, ok